.dockerignore

```diff
@@ -1,12 +1,41 @@
+# git history is big, do not add it
 .git
-.env
-tmp
-log
+
+# tests are big and not needed
 test
-.bundle
-config/database.yml
-cached_repos
-node_modules
+
+# user might chose to have local gem installs, they can be quiet big
 vendor/bundle
 
-public/assets
+# editor-specific folders that can be big
+.idea
+
+## copied from .gitignore
```

Comment:
doesn't dockerconfig support including the gitignore? why copy it?

Reply:
nope

Reply:
I must be thinking of the gcloud ignore...